### PR TITLE
tweak reset height when keyboard disapears #5502

### LINKED ISSUE #5502: Fix bottom inset issue for tokens screen

### PR DESCRIPTION
**Pull request by oa-s**
Fixes #5502

### CODE REVIEW

**Comment by oa-s** [comment]
Before pr


https://user-images.githubusercontent.com/96283950/193812815-53fd6a39-f2b7-47da-af98-f64ba51cbd35.mov

**Comment by oa-s** [comment]
After pr:

https://user-images.githubusercontent.com/96283950/193813228-ff6e0d6c-869f-4006-aa39-ba44457c71de.mov

File: AlphaWallet/Common/Services/KeyboardChecker.swift [hunk 1]
```diff
@@ -31,6 +31,8 @@ class KeyboardChecker: NSObject {
     }
 
     @objc func viewWillAppear() {
+        viewWillDisappear()
```
Comment (hboon) [inline AlphaWallet/Common/Services/KeyboardChecker.swift:34]: Haha. Doesn't look right :)

**Review by hboon — APPROVED** [review]
Works except for that strange `viewWillDisappear()` call. Good to merge after fixing?

**Pull request merged**